### ISSUE TITLE
OPTIONS[`string_if_invalid`] in settings.TEMPLATES breaks functionality

### ISSUE THREAD

**Issue by alxs** [theatlantic/django-nested-admin#70]
Wanting to have nested inlines in the admin (without sorting), I tried the app with a string value for `OPTIONS['string_if_invalid']` in `settings.TEMPLATES`. I encountered two errors:

- Without assigning a `sortable_field_name` to the inline, a `KeyError` was raised, because the `string_if_invalid` value is assumed to be a field of the `ModelForm` used, which was not the case.

- I therefore set `sortable_field_name` to `None` (assuming it would disable sorting). In this case, the `string_if_invalid` value was displayed in the HTML and the formset was not working at all.

Without setting `string_if_invalid'`, the app works, but still I think this is not an expected behavior, unless `sortable_field_name` is required, which I did not see mentioned in the docs (perhaps I missed it?).

**Comment by fdintino** [comment]
It's not required, I think you've encountered a bug, as I had not considered the consequences of `string_if_invalid`. What happens if you set `sortable_field_name` to an empty string? If that works then I would use that as a workaround until I get a chance to fix the issue.

Also, as I'll want to write a regression test, could you share what version of Django you are using, as well as whether you have either django-grappelli or django-suit installed?

Thanks.

**Comment by alxs** [comment]
Setting `string_if_invalid` to an empty string has the same effect as setting it to `None`.

I am using django 1.10.5 and I use neither django-grappelli nor django-suit. I have done some minor styling changes with CSS, but even without them the result is the same. 

I actually tested this on another project as well. When I set the `string_if_invalid` to a string value, the result was the same with the one I mentioned before.

So yeah, it seems to be a bug.

Thanks
